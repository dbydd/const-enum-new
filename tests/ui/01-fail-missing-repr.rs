#![feature(const_trait_impl)]

use const_enum::ConstEnum;

#[derive(ConstEnum)]
enum Test {
    A = 1,
    B = 2,
    C = 3,
}

pub fn main() {}
