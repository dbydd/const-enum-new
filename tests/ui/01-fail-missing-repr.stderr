error: proc-macro derive panicked
 --> tests/ui/01-fail-missing-repr.rs:5:10
  |
5 | #[derive(ConstEnum)]
  |          ^^^^^^^^^
  |
  = help: message: repr attribute not found on enum
